test/e2e/playwright/global/specs/protect-intrinsics.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from '@playwright/test';
```

Comment:
hi there :wave: I have a couple of questions regarding this e2e implementation:
- is there any reason why we are using playwright?
- we have a lockdown e2e spec [here](https://github.com/MetaMask/metamask-extension/blob/develop/test/e2e/tests/account/lockdown.spec.js). I'm wondering what's the difference with this one, as it looks like it's already achieving the same goal? :thinking:

Reply:
Hi!
- No reason from my side. Is there any reason we should not be using playwright? I just used it, because I created this PR based on an example that @Gudahtt gave me: https://github.com/MetaMask/browser-passworder/blob/main/test/index.spec.ts, and I saw some other tests also using playwright.
- That test indeed looks pretty similar, I did not know such a test already existed. Maybe @Gudahtt has a better understanding of the differences? I guess in this case we can reduce the PR to just deleting the unit test.